src/components/Footer.tsx

```diff
@@ -0,0 +1,84 @@
+import Image from "next/image";
+import Link from "next/link";
+
+function List({ text, children }: { text: string; children: React.ReactNode }) {
+  return (
+    <div>
+      <h2 className="mb-5 text-base font-medium">{text}</h2>
+      <ul>{children}</ul>
+    </div>
+  );
+}
+
+function ListItem({
+  url,
+  target = "_blank",
+  text,
+}: {
+  url: string;
+  target?: string;
+  text: string;
+}) {
+  return (
+    <li className="mb-2 text-sm">
+      <Link href={url} target={target}>
+        {text}
+      </Link>
+    </li>
+  );
+}
+
+function Footer() {
+  return (
+    <>
+      <div className="grid place-items-center md:mb-16 md:mt-24">
+        <Image
+          src="/Juwe2025.png"
+          alt="Description of the image"
+          className="my-20 md:hidden"
+          width={175}
+          height={175}
+        />
+        <div className="grid grid-cols-2 gap-x-5 sm:grid-cols-4 sm:justify-around md:flex md:w-11/12 md:flex-row md:justify-around">
```

Comment:
Zwieksz odleglosci na moblice
```suggestion
        <div className="grid grid-cols-2 gap-x-5  gap-y-10 sm:grid-cols-4 sm:justify-around md:flex md:w-11/12 md:flex-row md:justify-around">
```